docs/backends/lua2.rst

```diff
@@ -0,0 +1,173 @@
+Lua Backend
+===========
+
+* Native: Yes
+* Master: Yes
+* Slave: No
+* Superslave: No
+* Autoserial: No
+* DNSSEC: Yes
+* Disabled data: No
+* Comments: No
+* Module name: lua2
+* Launch name: ``lua2``
+
+This is a rewrite of existing Lua backend.
+This backend is stub between your Lua script and PowerDNS authoritative server.
+The backend uses AuthLua4 base class, and you can use same functions and types as in any other Lua script.
+
+.. warning::
+  Some of the function calls and configuration settings have been changed, please review this document carefully.
+
+.. warning::
+  All places which use DNS names now use DNSName class which cannot be compared directly to a string.
+  To compare them against a string use either ``tostring(dnsname)`` or ``newDN(string)``.
+
+API description (v1)
```

Comment:
maybe call this v2?

Reply:
Good question, not sure. I'm happy with any way you decide. =)

Reply:
Let's go with v2 to avoid any confusion.

Reply:
Changed. Also added check that if version 1 is used, it instructs to use luabackend instead.